Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -31,8 +31,12 @@ def __init__(self, message_bytes: bytes, include_raw_body: bool, save_file: bool
                          '\nWill replace backslash and try to parse again')
             message_bytes = self.handle_message_slashes(message_bytes)
             email_object = parse_from_bytes(message_bytes)
-        except Exception:
-            email_object = parse_from_string(message_bytes.decode('ISO-8859-1'))
+        except TypeError as e:
```

Comment:
is there a possibility that a different exception will be raised here and not handled correctly?
if so, the fetch can potentially get stuck.